avocado/core/test.py

```diff
@@ -1019,6 +1019,8 @@ class SimpleTest(Test):
     Run an arbitrary command that returns either 0 (PASS) or !=0 (FAIL).
     """
 
+    DATA_SOURCES = ["file", "variant"]
```

Comment:
Again as mentioned earlier I don't think an extra data source is a big problem, but missing `test` data source could bring confusion to people using custom class inherited from `SimpleTest` with multiple `test` methods (eg. some diff SimpleTest, performance SimpleTest etc.).

Reply:
They, since they'll be using another class, they can revert the original sources, or even add their own.